openfisca_core/formulas.py

```diff
@@ -663,6 +620,50 @@ def get_neutralized_column(column):
         )
 
 
+def extract_formula_date_from_name(formula_name, formula_name_prefix, formula_name_separator):
+    """
+    Return a day date extracted from formula name.
+    Valid dated name formats are : 'formula', 'formula_YYYY', 'formula_YYYY_MM' and 'formula_YYYY_MM_DD' where YYYY, MM and DD are a year, month and day.
+
+    Default year is '0001'. Default month and day are '01'.
+    """
+    formula_default_year = '0001'
+    formula_default_month = '01'
+    formula_default_day = '01'
+
+    if formula_name == formula_name_prefix:
+        formula_name += formula_name_separator + formula_default_year + formula_name_separator\
```

Comment:
`W503` is [highly controversial](https://gitlab.com/pycqa/flake8/issues/139) and [outdated](https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator).

I'm black-listing it right now.